CMakeLists.txt

```diff
@@ -69,6 +69,7 @@ elseif(WIN32)
   include(Win32Configuration)
 elseif(UNIX AND (NOT APPLE))
```

Comment:
We'd probably require a NOT OPENELEC test here as well as the desktop shortcut will be useless there.

Reply:
this is not needed (seem comment below)